src/bin/pg_dump/pg_dumpall.c

```diff
@@ -1212,16 +1234,19 @@ dumpRoles(PGconn *conn)
 	/* note: rolconfig is dumped later */
 	if (server_version >= 90600)
 		printfPQExpBuffer(buf,
-						  "SELECT oid, rolname, rolsuper, rolinherit, "
+						  "SELECT %s.oid, rolname, rolsuper, rolinherit, "
 						  "rolcreaterole, rolcreatedb, "
 						  "rolcanlogin, rolconnlimit, rolpassword, "
```

Comment:
not server_version >= 9600, we are PG14 later

Reply:
OK. I will change that.